client/landing/jetpack-cloud/components/daily-backup-status/index.jsx

```diff
@@ -36,17 +36,21 @@ class DailyBackupStatus extends Component {
 		const { allowRestore, backupAttempts, moment, translate } = this.props;
 
 		const displayDate = moment( backupAttempts.complete[ 0 ].activityDate ).format(
-			'MMMM Do YYYY, h:mm:ss a'
+			'MMMM Do YYYY, h:mm a'
 		);
 
 		return (
 			<Fragment>
-				<Gridicon icon="cloud-upload" />
+				<Gridicon className="daily-backup-status__status-icon" icon="cloud-upload" />
 				<div className="daily-backup-status__label">
-					{ translate( 'Latest backup completed:' ) }
+					<Fragment>{ translate( 'Latest: ' ) }</Fragment>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'Latest', { context: 'latest blog posts'} )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/jetpack-cloud-backup-status-design)